rpc/src/server/mod.rs

```diff
@@ -128,11 +128,11 @@ pub type HResult = Result<Response<Body>, Box<dyn std::error::Error + Sync + Sen
 
 pub type Handler = Arc<
     dyn Fn(
-        Request<Body>,
-        Params,
-        Query,
-        RpcServiceEnvironment,
-    ) -> Box<dyn Future<Output = HResult> + Send>
+            Request<Body>,
```

Comment:
@sebastiencs about this fmt, I don't know why, but locally my `cargo fmt` does this here, but then the checker in GitHub actions complains that it is wrong (and the old version here is the correct one). Please revert it.

I don't know why it behaves differently, but it is super annoying.